### PR TITLE
INTMDB-252: Added two parameters for cluster advanced configuration

### PR DESCRIPTION
**Pull request by coderGo93**
## Description
- Added two parameters called `default_read_concern` and `default_write_concern` for cluster under `advanced_configuration`
- Added test `TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf`
- Passed nil for create a project

Results of `TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf`

```
 make testacc TESTARGS='-run=TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... | grep -v /integrationtesting) -v -parallel 20 -run=TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf -timeout 300m -cover -ldflags="-s -w -X 'github.com/mongodb/terraform-provider-mongodbatlas/version.ProviderVersion=acc'"
?       github.com/mongodb/terraform-provider-mongodbatlas      [no test files]
=== RUN   TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf
=== PAUSE TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf
=== CONT  TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf
--- PASS: TestAccResourceMongoDBAtlasCluster_basic_DefaultWriteRead_AdvancedConf (767.71s)
PASS
coverage: 8.2% of statements
ok      github.com/mongodb/terraform-provider-mongodbatlas/mongodbatlas 768.020s        coverage: 8.2% of statements
?       github.com/mongodb/terraform-provider-mongodbatlas/version      [no test files]

```

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_cluster_test.go [hunk 1]
```diff
@@ -1199,6 +1258,48 @@ func testAccMongoDBAtlasClusterConfigAdvancedConf(projectID, name, autoscalingEn
...
+
+			advanced_configuration  {
+				default_read_concern                 = %[10]q
+				default_write_concern                = %[11]q
+				javascript_enabled                   = %[4]t
+				minimum_enabled_tls_protocol         = %[5]q
+				no_table_scan                        = %[6]t
+				oplog_size_mb                        = %[7]d
+				sample_size_bi_connector			 = %[8]d
+				sample_refresh_interval_bi_connector = %[9]d
```
Comment (thetonymaster) [inline mongodbatlas/resource_mongodbatlas_cluster_test.go]: Align this line

**Review by thetonymaster — APPROVED** [review]

File: mongodbatlas/resource_mongodbatlas_cluster_test.go [hunk 2]
```diff
@@ -1233,6 +1334,40 @@ func testAccMongoDBAtlasClusterConfigAdvancedConfPartial(projectID, name, autosc
...
+            cluster_type = "REPLICASET"
+		    replication_specs {
+			  num_shards = 1
+			  regions_config {
+			     region_name     = "EU_CENTRAL_1"
+			     electable_nodes = 3
+			     priority        = 7
+                 read_only_nodes = 0
+		       }
+		    }
```
Comment (andreaangiolillo) [inline mongodbatlas/resource_mongodbatlas_cluster_test.go]: [q] Could you align this piece of code?

File: mongodbatlas/resource_mongodbatlas_cluster_test.go [hunk 3]
```diff
@@ -1199,6 +1258,48 @@ func testAccMongoDBAtlasClusterConfigAdvancedConf(projectID, name, autoscalingEn
...
+			provider_instance_size_name = "M10"
+
+			advanced_configuration  {
+				default_read_concern                 = %[10]q
+				default_write_concern                = %[11]q
+				javascript_enabled                   = %[4]t
+				minimum_enabled_tls_protocol         = %[5]q
+				no_table_scan                        = %[6]t
+				oplog_size_mb                        = %[7]d
+				sample_size_bi_connector			 = %[8]d
```
Comment (andreaangiolillo) [inline mongodbatlas/resource_mongodbatlas_cluster_test.go]: [q] Could you align this line?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: mongodbatlas/resource_mongodbatlas_cluster_test.go (hunk 2 above)
Comment (coderGo93) [reply mongodbatlas/resource_mongodbatlas_cluster_test.go]: aligned , thank you

File: mongodbatlas/resource_mongodbatlas_cluster_test.go (hunk 3 above)
Comment (coderGo93) [reply mongodbatlas/resource_mongodbatlas_cluster_test.go]: aligned , thank you

File: mongodbatlas/resource_mongodbatlas_cluster_test.go (hunk 1 above)
Comment (coderGo93) [reply mongodbatlas/resource_mongodbatlas_cluster_test.go]: aligned , thank you

**Pull request merged**